cmd/demo/node_setup.go

```diff
@@ -52,9 +52,9 @@ func Setup() {
 }
 
 func newNode() (*node, error) {
-	wallet, acc, err := importAccount(config.SecretKey)
+	wallet, acc, err := importMnemonic(config.Mnemonic, config.AccountIndex)
```

Comment:
You renamed the function from `importAccount` to `importMnemonic` but it still imports an `Account` :laughing: Cou could also name it `setupWallet` or `createWallet`.